PSHTML/Functions/Public/sub.ps1

```diff
@@ -1,89 +1,81 @@
 Function SUB {
     <#
     .SYNOPSIS
-    Create a SUB title in an HTML document.
-
+        Create a SUB tag in an HTML document.
+    .DESCRIPTION
+        Create a SUB tag in an HTML document. 
     .EXAMPLE
-
-    SUB
-    .EXAMPLE
-    SUB "woop1" -Class "class"
-
-    .EXAMPLE
-    SUB "woop2" -Class "class" -Id "MainTitle"
-
-    .EXAMPLE
-    SUB {"woop3"} -Class "class" -Id "MaintTitle" -Style "color:red;"
-
-    .Notes
-    Author: Chendrayan Venkatesan (Chen V)
-    Version: 1.0.0
-    History:
-        2018.10.17;@ChendrayanV; New Version 1.0.0
+        p -content {
```

Comment:
It would be great to have also the HTML code that is generated here as an example.